test/driver/operator/cli_driver.go

```diff
@@ -183,9 +183,9 @@ type respTxQuery struct {
 	} `json:"tx_result"`
 }
 
-func (o *OperatorCLIDriver) SubmitMigrationProposal(ctx context.Context, activationHeight, migrationDuration *big.Int, chainID string) ([]byte, error) {
+func (o *OperatorCLIDriver) SubmitMigrationProposal(ctx context.Context, activationHeight, migrationDuration *big.Int) ([]byte, error) {
 	var res display.TxHashResponse
-	err := o.runCommand(ctx, &res, "migrate", "propose", activationHeight.String(), migrationDuration.String(), chainID)
+	err := o.runCommand(ctx, &res, "migrate", "propose", "activation-period", activationHeight.String(), "duration", migrationDuration.String())
```

Comment:
This doesn't need `--` on `activation-period`?

Reply:
Oh whoops. Currently we do not test the cli for this, but yes you are correct